x-pack/plugins/monitoring/server/lib/elasticsearch/nodes/get_nodes/__test__/get_node_ids.test.js

```diff
@@ -47,7 +47,7 @@ describe('getNodeIds', () => {
     };
     const clusterUuid = '1cb';
 
-    const result = await getNodeIds(req, '.monitoring-es-*', { clusterUuid }, 10);
+    const result = await getNodeIds(req, '.monitoring-es-*,monitoring-es-*', { clusterUuid }, 10);
```

Comment:
nit: Maybe some bad merge resolves here? Since, you don't need these none-system indices anymore because of: https://github.com/elastic/kibana/pull/56807 In couple of other places below too

Reply:
Yea, nice find.